x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -682,6 +677,18 @@ class PackagePolicyService {
       }
     }
   }
+
+  public async runDeleteExternalCallbacks(
+    deletedPackagePolicies: DeletePackagePoliciesResponse
+  ): Promise<void> {
+    const externalCallbacks = appContextService.getExternalCallbacks('postPackagePolicyDelete');
+
+    if (externalCallbacks && externalCallbacks.size > 0) {
+      for (const callback of externalCallbacks) {
+        await callback(deletedPackagePolicies);
```

Comment:
I'm thinking this here:

A failure in an external callback (ex. it `throw`'s), should **not** prevent this loop from calling other registered callbacks. Multiple kibana plugins might have registered callbacks for this and we probably want to allow them to also run instead of not calling them at all.

I will make that change unless there are objections 😃 

@nchaulet  ☝️